backend/internal/constants.go

```diff
@@ -0,0 +1,7 @@
+package internal
+
+// Blockchain-related constants
+const (
+	// SS58AddressFormat is the format used for Substrate-based chain addresses
+	SS58AddressFormat = 42
```

Comment:
rename the file into keypairs.go 
keep the const
move the code mentioned here https://github.com/codescalers/kubecloud/pull/85/files#r2219586657 to it

Reply:
Link doesn't redirect to specific code.

<img width="1845" height="1005" alt="image" src="https://github.com/user-attachments/assets/672657e5-5aa9-4c76-aee5-8b7d11a438ca" />